src/test/java/net/sf/jabref/logic/bibtex/BibEntryWriterTest.java

```diff
@@ -117,14 +117,8 @@ public void roundTripWithPrependingNewlines() throws IOException {
         ParserResult result = BibtexParser.parse(new StringReader(bibtexEntry));
 
         Collection<BibEntry> entries = result.getDatabase().getEntries();
-        assertEquals(1, entries.size());
 
         BibEntry entry = entries.iterator().next();
-        assertEquals("test", entry.getCiteKey());
```

Comment:
Are you being serious? In this very same PR you asked me to reduce the number asserts as far as possible, aiming for one assertion per test. This is the reason.